components/ItemCard.tsx

```diff
@@ -43,6 +44,12 @@ export const ItemCard: React.FC<ItemCardProps> = ({ holderId, ...item }) => {
     equippers.length > 0 &&
     equippers.some(equippedBy => equippedBy.characterId === holderId);
 
+  const requiredXp = useMemo(() => {
+    return BigInt(
+      requirements.filter(r => r.assetCategory === 'ERC20')[0]?.amount ?? '0',
```

Comment:
Hey we also need to check if the assetAddress is the experience token. In all the places.